opennms-model/src/main/java/org/opennms/netmgt/model/OnmsMonitoredService.java

```diff
@@ -614,4 +620,22 @@ public String getForeignId() {
         }
         return null;
     }
+
+    @Transient
+    @XmlTransient
+    @JsonIgnore
+    public Set<String> getReductionKeys() {
```

Comment:
Can we encapsulate the reduction key logic in one of the BSM-specific modules instead of adding it here?